### DISCUSSION TITLE
Tracer on Scheduler Tasks - Spring boot

### DISCUSSION THREAD

**Discussion by groversi** [jaegertracing/jaeger#3489] [Q&A]
I'm trying to instrument Jaeger on spring app, with a scheduler. But the behavior of Tracer is little bit strange from an app with api's. Jaeger does not create a new Trace when the scheduler starts, Tracer.activeSpan() returns null, and the uber-trace-id is not propagated to other services.

Is there anyone who did an instrumentation on a Spring scheduler?

**Comment by pavolloffay** [comment]
This is instrumentation-related question, please ask the question in the appropriate instrumentation repository.
